### PR TITLE
Refactor pipelines to use common steps

### PR DESCRIPTION
**Pull request by fritze2**
- Import modularized steps from `.ci-orchestrator/steps` into `.ci-orchestrator/pb.yml`.

### CODE REVIEW

**Comment by fritze2** [comment]
#pbbeta

**Comment by fritze2** [comment]
#fullpbbeta

File: .ci-orchestrator/steps/fats/liteCheckpointFATs.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Checkpoint FATs therefore run twice in builds but the @CheckpointTest annotation and use of the fat.test.run.checkpoint.only property mean that in CRIU environments
+# they only run the test cases with @CheckpointTest (skipping the others) and in non-CRIU environments they only run the unannotated test cases. Hence, there's no duplication of testing.
+- stepName: Checkpoint FATs (Lite)
+  conditionalRun:
+    - type: ifTrue
+      value: ${spawn.checkpoint}
+  workType: FAT
+  timeoutInMinutes: 1920
+  properties:
+    spawn.checkpoint: true
```
Comment (wraschke) [inline .ci-orchestrator/steps/fats/liteCheckpointFATs.yml:11]: I don't know if this is important enough to warrant a change, but in other files with `spawn.checkpoint` property, its value is `"true"` (i.e., no surrounding quotation marks here).

File: .ci-orchestrator/steps/fats/zOSUnittests.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    # Product to be tested.
+    product_image_type_under_test: wlp-zosimage
+
+    # Buckets to be tested.
+    fat.buckets.to.run: ${PR Changes:fat.buckets.to.run}
+    fat.test.mode: lite
+    # Pattern causes us to only execute the z/OS unittests.
+    fatPatternToMatch: .*_ztest
+    runZosTests: true
+    command: ant -f build-zunittest.xml unittest -propertyfile ../../../buildandbucket.properties -DhaltOnFailure=false -lib ../ant_build/lib/ecj-4.3.1.jar -lib ../ant_build/lib.antClasspath -lib ../prereq.published/lib -lib ../ant_build/lib/biz.aQute.bnd-3.3.0.jar -lib ../ant_build/lib/jsoup-1.7.2.jar
```
Comment (wraschke) [inline .ci-orchestrator/steps/fats/zOSUnittests.yml:22]: If anyone ever upgrades these files' versions with intention of replacing the files mentioned here, we need to be sure that their test pipelines run the command with those new files.

**Review by wraschke — APPROVED** [review]
I've looked at the pbbeta and fullpbbeta pipelines but not triaged the failures. I see they have some unmatched failures and you will triage/find matches and investigate before merging, yes?

**Comment by fritze2** [comment]
Took a look at the untriaged failures and don't think they are related to my change. All failures are intermittent. I see one failure that doesn't have a close match, but is failing in a bucket where the other failures with matches occurred. The other untriaged failures had matches and I triaged them. I do think it's safe to merge.

**Comment by fritze2** [comment]
#libby

**Pull request merged**